R/PipeOpDateFeatures.R

```diff
@@ -0,0 +1,209 @@
+#' @title PipeOpDateFeatures
+#'
+#' @usage NULL
+#' @name mlr_pipeops_datefeaturs
+#' @format [`R6Class`] object inheriting from [`PipeOpTaskPreprocSimple`]/[`PipeOp`].
+#'
+#' @description
+#' Based on a specified `POSIXct` column of the data, a set of date related features is computed and
+#' added to the feature set of the output task.
+#'
```

Comment:
Please add a reference to the original source code from fast.ai

Reply:
fixed